tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -197,6 +198,17 @@ def databricks_sql_ds(context: EphemeralDataContext) -> DatabricksSQLDatasource:
     return ds
 
 
+def snowflake_creds_populated():
+    if os.getenv("SNOWFLAKE_CI_USER_PASSWORD") in [None, ""] or os.getenv(
+        "SNOWFLAKE_CI_ACCOUNT"
+    ) in [None, ""]:
+        return False
+    return True
```

Comment:
```suggestion
    if os.getenv("SNOWFLAKE_CI_USER_PASSWORD") or os.getenv(
        "SNOWFLAKE_CI_ACCOUNT"
    ):
        return True
    return False
```
Python's built-in truthy evaluation will give you want you want here.

Reply:
Thank you @Kilo59